utilities/pipelines/resourcePublish/Publish-ModuleToUniversalArtifactFeed.ps1

```diff
@@ -0,0 +1,118 @@
+<#
+.SYNOPSIS
+Publish a new version of a given module to a DevOps artifact feed
+
+.DESCRIPTION
+Publish a new version of a given module to a DevOps artifact feed
+
+.PARAMETER TemplateFilePath
+Mandatory. Path to the module deployment file from root.
+Example: 'C:\arm\Microsoft.KeyVault\vaults\deploy.bicep'
+
+.PARAMETER ModuleVersion
+Required. Version of the module to publish, following SemVer convention.
```

Comment:
Minor though I'd choose between "Mandatory" and "Required"

Reply:
Fixed an aligned on other scripts in this PR. 

Documentation: 'Mandatory' -> Needs to be aligned on more ps1 files, as others also use "Required." As this is automatically identified and documented by PowerShell, why document it in description?

Implementation:
- 'Mandatory' for when the parameter is needed
- 'Mandatory=$false' when the parameter is not needed